ngraph_bridge/enable_variable_ops/ngraph_replace_variable_modifiers.cc

```diff
@@ -35,6 +35,33 @@ namespace ngraph_bridge {
 // 1. ReplaceModifiers
 // 2. RewriteForTracking
 
+template <typename T1, typename T2>
+Status CreateBinaryOpNode(NodeBuilder::NodeOut* ndef_gen_op,
+                          std::string new_name_op, std::string op_type,
+                          T1 input1, T2 input2, DataType dtype, Node*& node,
+                          Graph*& graph, bool replace_input,
+                          bool replace_output) {
+  Node* generic_op;
+  new_name_op = node->name() + new_name_op;
+
+  TF_RETURN_IF_ERROR(NodeBuilder(new_name_op, op_type)
+                         .Input(input1)
+                         .Input(input2)
+                         .Attr("T", dtype)
+                         .Device(node->assigned_device_name())
+                         .Finalize(graph, &(generic_op)));
+  generic_op->set_assigned_device_name(node->assigned_device_name());
+
+  if (replace_input) {
+    ReplaceInputControlEdges(graph, node, generic_op);
```

Comment:
`ReplaceInputControlEdges` and `ReplaceOutputEdges` return `Status`es, hence they need to be wrapped by a `TF_RETURN_IF_ERROR`